docs/en/observability/synthetics-run-test.asciidoc

```diff
@@ -0,0 +1,205 @@
+* If you have multiple journeys, rely on dependencies, or your tests need to live with your
+application code, <<synthetics-test-suite,use a test suite>>.
+
+[discrete]
+[[synthetics-inline-journey]]
+== Use an inline journey
+
+The easiest way to run a synthetic test is by creating an inline journey.
+The `journey` keyword isn't required, and access to variables like `page` and `params` is automatic.
+You cannot `import` any dependencies using inline scripts.
+
+To test an inline example locally, change into your test's directory
+and pipe the file contents to the `npx @elastic/synthetics` command.
+
+For example, create a `sample.js` file containing steps:
+
+[source,js]
+----
+step('load homepage', async () => {
+    await page.goto('https://www.elastic.co');
+});
+step('hover over products menu', async () => {
+    await page.hover('css=[data-nav-item=products]');
+});
+----
+
+Then test the sample file:
+
+[source,sh]
+----
+cat path/to/sample.js | npx @elastic/synthetics --inline
+----
+
+And you'll get a response similar to:
+
+[source,sh]
+----
+Journey: inline
+   ✓  Step: 'load homepage' succeeded (1831 ms)
+   ✓  Step: 'hover over products menu' succeeded (97 ms)
+
+ 2 passed (2511 ms)
+----
+
+After testing locally, you can copy the script into your
+Elastic Synthetics integration or `heartbeat.yml`.
+
+* **{agent} and {fleet}**: Add an inline journey when configuring the Elastic Synthetics integration.
+In Monitor settings, use Monitor Type "Browser" and switch the Source Type to "Inline script".
+See <<uptime-set-up>> for details.
+* **{heartbeat}**: Copy and paste your test steps into `heartbeat.yml`.
+{heartbeat} spawns a separate Node.js process, schedules your tests, and runs them on a Chromium browser.
+See <<uptime-set-up-config>> for details.
+
+// heartbeat
+//
+// [source,yml]
+// ----
+// - type: browser
+//   id: elastic-website <1>
+//   name: Elastic website
+//   schedule: "@every 1m"
+//   source:
+//     inline:
+//       script: |- <2>
+//       step("load homepage", async () => {
+//         await page.goto('https://www.elastic.co');
+//       });
+//       step("hover over products menu", async () => {
+//         await page.hover('css=[data-nav-item=products]');
+//       });
+// ----
+// <1> Each `monitor` gets its own ID in the {uptime-app} and, therefore its own schedule entry.
+// This allows tests to be run in parallel and analyzed separately.
+// <2> In this example, a synthetic test is defined inline. This is a two-step script that first loads
+// a homepage and then hovers over a product menu. See <<synthetics-syntax>> for more information.
+
+
+[discrete]
+[[synthetics-test-suite]]
+== Use a test suite
+
+If you have a suite of tests to implement, you can use Elastic Synthetics as a library.
+In this method, you use Docker to run both {heartbeat} and `elastic-synthetics`.
+
+To test an inline example locally, start by installing the `@elastic/synthetics` package globally:
```

Comment:
I'd follow the inverse order here. I'd first tell users to write the tests, and only when the package is necessary I'd tell them to install it. Otherwise, they'll have downloaded something they'll only use in later steps.

Also, I think we should not say "to test an inline example locally" as this tutorial is about suites.